VideoLocker/res/layout/fragment_my_all_videos.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Since we aren't using data binding in the fragment, there's no _need_ to do so in this PR. Though I'm not sure there's any harm in doing it.